query/src/main/scala/filodb/query/exec/RangeVectorTransformer.scala

```diff
@@ -210,4 +211,36 @@ final case class MiscellaneousFunctionMapper(function: MiscellaneousFunctionId,
             sourceSchema: ResultSchema): Observable[RangeVector] = {
     miscFunction.execute(source)
   }
+}
+
+final case class SortFunctionMapper(function: SortFunctionId) extends RangeVectorTransformer  {
+  protected[exec] def args: String =
+    s"function=$function"
+
+  def apply(dataset: Dataset,
+            source: Observable[RangeVector],
+            queryConfig: QueryConfig,
+            limit: Int,
+            sourceSchema: ResultSchema): Observable[RangeVector] = {
+
+    val ordering: Ordering[Double] = function match {
```

Comment:
Please add a check that the resultSchema output is in fact a Double, otherwise the error message will be difficult.  
If the result schema is not a Double, for now maybe just pass the input unchanged (though histograms can also be compared).

Reply:
Added check for schema column to be double